wipasswd.swift

```diff
@@ -52,19 +60,24 @@ func main() {
     }
 
     if let s = ssid {
-        let contentsOfKeychain = getPasswd(s)
-        if let pass = contentsOfKeychain {
-            println("SSID: \(s)")
-            println("PASS: \(pass)")
-            exit(1)
+        if let pass = getPasswd(userAccount: s) {
+            // Copy to clipboard
+            let pb = NSPasteboard.general()
+            pb.clearContents()
+            pb.setString(pass, forType:NSStringPboardType)
```

Comment:
nice and neat change to implement copy-to-clipboard. But, I'd like to have a different commit from this set of changes for Swift3